### PR TITLE
101-vm-simple-linux-with-accelerated-networking

### PR DESCRIPTION
**Pull request by FabienLavocat**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [X] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

* Create template for simple VM with Accelerated Networking

### CODE REVIEW

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    "storageAccountKind": "Storage",
+
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+
+    "vmName": "MyUbuntuVM",
+    "vmSize": "Standard_D3_v2",
+
+    "virtualNetworkName": "MyVNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
```
Comment (bmoore-msft) [inline 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: remove this and just use resourceId for the subnet ref below...

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        "tier": "[variables('storageAccountSkuType')]"
+      },
+      "kind": "[variables('storageAccountKind')]",
+      "properties": {}
+    },
+
+    {
+      "type": "Microsoft.Network/publicIPAddresses",
+      "name": "[variables('publicIPAddressName')]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: update apiVersion

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    "virtualNetworkName": "MyVNET",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetID'),'/subnets/',variables('subnetName'))]"
+  },
+  "resources": [
+
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('storageAccountName')]",
+      "apiVersion": "2016-01-01",
```
Comment (bmoore-msft) [inline 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: update apiVersion

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses',variables('publicIPAddressName'))]"
+              },
+              "subnet": {
+                "id": "[variables('subnetRef')]"
+              }
+            }
+          }
+        ]
+      },
+      "dependsOn": [
```
Comment (bmoore-msft) [inline 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: move dependsOn above properties

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.parameters.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "value": "ghuser"
```
Comment (bmoore-msft) [inline 101-vm-simple-linux-with-accelerated-networking/azuredeploy.parameters.json]: GEN-UNIQUE

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json (hunk 3 above)
Comment (FabienLavocat) [reply 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: I'm updating to 2017-06-01

File: 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json (hunk 2 above)
Comment (FabienLavocat) [reply 101-vm-simple-linux-with-accelerated-networking/azuredeploy.json]: I'm updating to 2017-06-01

**Pull request merged**